src/atoms/Text/Text.native.js

```diff
@@ -32,6 +32,12 @@ const styles = {
       return getLineHeight(theme, size);
     }
   },
+  textAlign({ align }) {
+    if (align == 'inherit' || align == 'initial') {
+      return 'auto';
```

Comment:
Is this because React Native doesn't have `inherit` or `initial`?

Reply:
RN supports `'auto', 'left', 'right', 'center', 'justify' `